server/actions/ItemDefinitions.ts

```diff
@@ -0,0 +1,7 @@
+import mongoDb from '../index'
+import ItemDefinitionSchema from '../models/ItemDefinition'
+
+export async function getItemDefinitions() {
```

Comment:
After you merge with main and resolve the conflicts, move this function over to `server/actions/ItemDefinition.ts`. Even though it's for multiple item definitions, the pattern we've established is having all actions related to a server model in one typescript file